app/react/docker/images/ListView/ImagesDatatable/columns/tags.tsx

```diff
@@ -15,11 +15,12 @@ function Cell({ getValue }: CellContext<ImagesListResponse, string[]>) {
 
   return (
     <>
-      {repoTags.map((tag, idx) => (
-        <span key={idx} className="label label-primary image-tag" title={tag}>
-          {tag}
-        </span>
-      ))}
+      {repoTags &&
+        repoTags.map((tag, idx) => (
+          <span key={idx} className="label label-primary image-tag" title={tag}>
+            {tag}
+          </span>
+        ))}
```

Comment:
```suggestion
      {repoTags?.map((tag, idx) => (
          <span key={idx} className="label label-primary image-tag" title={tag}>
            {tag}
          </span>
        ))}
```

Reply:
if repoTags can be null, we should state that in https://github.com/portainer/portainer/blob/9bf2957ea7a2d12a583eb5731064744838fe9dc3/app/react/docker/images/queries/useImages.ts#L14

also, I'm pretty sure this pr makes `image.Tags` always not null, so this PR is obsolete. although we might want to add it anyway.


https://github.com/portainer/portainer/commit/5c7f6aab66ced3c1011a40e21ff786d15feba1a1